server/src/db/model.ts

```diff
@@ -21,6 +30,17 @@ const userSchema = new mongoose.Schema({
         height: { type: Number, required: true },
         fitnessLevel: { type: String, required: true },
     },
+    //using this field if the user is given the choice to skip the inetrest form
+    seen_greeting_modal: { type: Boolean, default: false },
```

Comment:
Change this to 'onboarded' instead of 'seen_greeting_modal'. Onboarding users in an app is a common concept, which sounds a little nicer (https://www.pendo.io/glossary/user-onboarding/)